Ellipse.ts

```diff
@@ -0,0 +1,155 @@
+    //     //top on y axis
+    //     points.push(new Point(this.center.x, this.center.y - this.radiusY));
+    //     //bottom on y axis
+    //     points.push(new Point(this.center.x, this.center.y + this.radiusY));
+    //     return points;
+    // }
+
+
+
+    // public onEllipse(otherEllipse: Ellipse) : boolean {
+    //     if(this.center.x <= otherEllipse.center.x){
+    //         //The other ellipse is on the right of this ellipse
+    //         if(otherEllipse.getBoundingBox()[1].x <= this.getBoundingBox()[2].x) {
+    //             //The other ellipse might be within the right half of this ellipse
+    //             if(this.center.y > otherEllipse.center.y) {
+    //                 //The other ellipse is above this ellipse
+    //                 if(otherEllipse.getBoundingBox()[4].y > this.getBoundingBox()[3].y){
+    //                     //other ellipse is within the the top right quadrant of this ellipse
+    //                     return true;
+    //                 }
+    //             } else {
+    //                 //The other ellipse is below this ellipse
+    //             }
+    //         } else {
+    //             //The center of the other ellipse might be within the left half of this ellipse
+    //         }
+    //     } else {
+    //         //The other ellipse is on the left of this ellipse
+    //     }
+    // }
+
+    // /**
+    //  * Accessor to get the center point of this ellipse.
+    //  * @returns The center point of this ellipse.
+    //  */
+    // public getCenter() : Point {
+    //     return this.center;
+    // }
+
+    // /**
+    //  * Modifier to set the center point of this ellipse.
+    //  * @param newCenter The point to be set as the new center of this ellipse.
+    //  */
+    // public setCenter(newCenter: Point) : void {
+    //     this.center = newCenter
+    // }
+
+    // /**
+    //  * Accessor to get the horizontal radius of this ellipse.
+    //  * @returns The horizontal radius of this ellipse.
+    //  */
+    // public getHorizontalRadius() : number {
+    //     return this.radiusX;
+    // }
+
+    // /**
+    //  * Modifier to set the horizontal radius of this ellipse.
+    //  * @param newRadX The length to be set as the new horizontal radius of this ellipse.
+    //  */
+    // public setHorizontalRadius(newRadX: number) : void {
+    //     this.radiusX = newRadX;
+    // }
+
+    // /**
+    //  * Accessor to get the vertical radius of this ellipse.
+    //  * @returns The vertical radius of this ellipse.
+    //  */
+    // public getVerticalRadius() : number {
+    //     return this.radiusY;
+    // }
+
+    // /**
+    //  * Modifier to set the vertical radius of this ellipse.
+    //  * @param newRadY The length to be set as the new vertical radius of this ellipse.
+    //  */
+    // public setVerticalRadius(newRadY: number) : void {
+    //     this.radiusY = newRadY
+    // }
```

Comment:
Dont submit commented out code for any final PRs that you want merged into main, if you ever need to look back at it, you can checkout the old versions of the code on github.